### PR TITLE
Separated Format

### PR DESCRIPTION
**Pull request by mirkof**
- Format separated into Writes and Reds
- project version updated
- versions of scala, play, scalates also updated

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -2,15 +2,15 @@ scalariformSettings
...
 name := """play-json-naming"""
 
-version := "0.1.0"
+version := "0.2.0"
 
-scalaVersion := "2.11.5"
+scalaVersion := "2.11.6"
 
-crossScalaVersions := Seq("2.10.4", "2.11.5")
+crossScalaVersions := Seq("2.10.4", "2.11.5", "2.11.6")
```
Comment (tototoshi) [inline build.sbt]: 2.11.5 can be removed

**Comment by mirkof** [comment]
is now everything ok? :)

**Comment by tototoshi** [comment]
Sorry to late reply. Could you change the version from 0.2.0 to 0.2.0-SNAPSHOT?

**Comment by mirkof** [comment]
Done.

**Comment by tototoshi** [comment]
Perfect :+1:

**Pull request merged**